KeenClient/KeenClient.m

```diff
@@ -243,37 +241,39 @@ + (void)disableGeoLocationDefaultRequest {
 
 
 + (void)clearAllEvents {
-    [dbStore deleteAllEvents];
+    [KIODBStore.sharedInstance deleteAllEvents];
 }
 
 + (void)clearAllQueries {
-    [dbStore deleteAllQueries];
+    [KIODBStore.sharedInstance deleteAllQueries];
 }
 
-+ (KIODBStore *) getDBStore {
-    return dbStore;
++ (KIODBStore *)getDBStore {
+    return KIODBStore.sharedInstance;
 }
 
-- (id)init {
+- (instancetype)init {
     self = [super init];
 
-    // log the current version number
-    if ([KeenClient isLoggingEnabled]) {
-        KCLog(@"KeenClient-iOS %@", kKeenSdkVersion);
-    }
+    if (nil != self) {
+        // log the current version number
+        if ([KeenClient isLoggingEnabled]) {
```

Comment:
Why do we check isLoggingEnabled here but not elsewhere. I thought KCLog() performed that check internally. PR #188 changes this a bit, but the check for logging enabled is still handled somewhere down in the logger I think, so it seems like either case this check can go away.